### PR TITLE
adds support for Ruby language

### PR DESCRIPTION
**Pull request by LeFnord**
BTW, it is tested on OS X with retina and non-retina display and it looks very good
… I like your colour scheme

### CODE REVIEW

File: styles/languages/ruby.less [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  }
+}
+
+.support {
+  &.class {
+    color: @zooce-yellow;
+  }
+
+  &.property-name {
+    color: @sand;
```
Comment (Zooce) [inline styles/languages/ruby.less:16]: "sand" needs to be defined

File: styles/languages/ruby.less [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  }
+
+  &.function  {
+    color: @butter;
```
Comment (Zooce) [inline styles/languages/ruby.less:20]: "butter" needs to be defined

File: styles/languages/ruby.less [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+  &.name.class, &.name.type.class {
+    color: @zooce-yellow;
+  }
+
+  &.name.section {
+    color: @zooce-yellow;
+  }
+
+  &.name.tag {
+    color: @indigo;
```
Comment (Zooce) [inline styles/languages/ruby.less:42]: "indigo" needs to be defined

File: styles/languages/ruby.less [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+  }
+
+  &.other.attribute-name {
+    color: @latte;
```
Comment (Zooce) [inline styles/languages/ruby.less:46]: "latte" needs to be defined

File: styles/languages/ruby.less [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+
+    &.id {
+      color: @fresh-blood;
```
Comment (Zooce) [inline styles/languages/ruby.less:49]: "fresh-blood" needs to be defined

File: styles/languages/ruby.less [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+  &.require {
+    color: @butter;
+  }
+
+  &.selector {
+    color: @butter;
+  }
+
+  &.separator {
+    background-color: @smoke;
```
Comment (Zooce) [inline styles/languages/ruby.less:72]: "smoke" needs to be defined

**Comment by LeFnord** [comment]
added missing colors in styles/colors.less

**Pull request closed without merging**